app/assets/stylesheets/_errors.scss

```diff
@@ -2,3 +2,12 @@
   color: $orange;
   font-style: italic;
 }
+
+form {
+  .error {
```

Comment:
I don't think we managed to develop platform for so long without implementing error styles. Isn't this somewhere else?

Reply:
I thought the same thing but I didn't find it.

The pattern used, is to list the errors at the top of the form. For example in `Patients`:

```
= form_for(@patient) do |f|
  - unless params[:next_url].blank?
    = hidden_field_tag :next_url, params[:next_url]

  = validation_errors @patient
```

Then `validation_errors` renders `partial: "shared/validation_errors"` but without any style (it shows as simple text)

I'm fixing it to this solution and then we may add some style to it.